docs/dev_contribution.rst

```diff
@@ -0,0 +1,95 @@
+
+For simple or obvious bugs which have fallen through the testing gaps just the implementation is fine.
+
+GitHub Pull Request
+===================
+
+Once some changes have been made and local commits committed push your changes to your GitHub libcellml repository (refer to :numref:`Figure %s <libcellml-branching>`).  From there create a pull request from your topic branch to the :term:`prime repository` develop branch.  When creating the pull request make sure to add in the comment 'addresses issue #46' (of course replace the number 46 with the actual number of the issue you are addressing), or something to that effect.  This will create a link between the issue and the pull request enabling other people to see that you are working on this issue and comment on your work.
+
+The following Git command line commands show an example of how to add all files, commit the changes and push them to a GitHub repository for the first time::
+
+   git add .
+   git commit -m "Appropiately descriptive message about the changes made."
+   git push -u origin iss46
+
+The 'git add' and 'git commit' commands should be obvious,  the 'git push' command sets the local branch 'iss46' to be linked with the remote branch 'iss46' in the origin (the default shorthand for your libCellML repository on GitHub) repository, this branch will be created in the origin repository if it doesn't already exist.  
+
+To create a pull request from one GitHub repository to another follow the instructions `here <https://help.github.com/articles/creating-a-pull-request/>`_.
+
+Satisfy Comments
+================
+
+It is important to respond to all feedback appropriately, the review process will check to make sure that all comments have been dealt with.  Feel free to respond to comments as appropriate, e.g. through code changes, posting a direct reply etc.
+
+Review
+======
+
+It may happen that submitted work is not reviewed immediately or the work is finished before any comments have been made.  If this is the case add a comment to the pull request asking for the submission to be reviewed.  An email will be sent out to the repository owners who will respond and review the submission, please remember that everyone is busy and it may not happen right away.
+
+Completion
+==========
+
+Once the owners of the :term:`prime repository` are satisfied that the work on the issue is complete and that the feedback has been addressed, one of the owners will post a comment on the issue notifying subscribers that they intend to merge the pull request.  If no further objections are raised the owner will merge an close the pull request as outlined in :doc:`Reveiw Process <dev_review>`.
```

Comment:
If I recall correctly, one of the owners must say that s/he is happy with the pull request. Then, a second owner must also say that s/he is happy. Once that is done, and because we have two 'happy' owners, the second owner can merge the pull request.

Minor point: ``` :doc:`Review Process <dev_review>` ``` and not ``` :doc:`Reveiw Process <dev_review>` ```.